deps/wazuh_testing/wazuh_testing/modules/vulnerability_detector/event_monitor.py

```diff
@@ -247,3 +251,80 @@ def check_vulnerability_scan_remove_alerts(wazuh_alerts_monitor, package, cve):
                                                               f"to a package removal/update or a system upgrade",
                               prefix='.*', error_message='Could not find the alert which says that the package '
                                                          f"{package} is no longer vulnerable with {cve}")
+
+
+def check_provider_database_update_start_log(provider_name):
+    """Check the provider database update start event in ossec.log
+
+    Args:
+        provider_name (str): Provider name of the downloaded feed.
+    """
+    check_vuln_detector_event(timeout=vd.VULN_DETECTOR_FAST_TIMEOUT,
+                              callback=f"Starting '{provider_name}' database update",
+                              error_message=f"Could not find {provider_name} update starting log")
+
+
+def check_provider_database_update_finish_log(provider_name):
+    """Check the provider database update finish event in ossec.log
+
+    Args:
+        provider_name (str): Provider name of the downloaded feed.
+    """
+    check_vuln_detector_event(timeout=vd.DOWNLOAD_TIMEOUT,
+                              callback=f"The update of the '{provider_name}' feed finished",
+                              error_message=f"Could not find {provider_name} feed finished log")
+
+
+def check_vulnerability_scan_remove_vuln_package(package, cve):
+    """Check if removed vulnerable packages are reported by vulnerability detector in the alerts.
+    Args:
+        package (str): Name of custom package to check. Example: 'firefox-0'
+        cve (str): Package CVE. Example: 'CVE-2019-11764'
+    """
+    check_vuln_detector_event(timeout=vd.VULN_DETECTOR_SCAN_TIMEOUT,
+                              update_position=False, callback=f"Package '{package}' not vulnerable to '{cve}'.",
+                              prefix='.*', error_message='Could not find the alert which says that the package '
```

Comment:
It is necessary to pass the `update_position`  and `prefix`?

Reply:
Solved https://github.com/wazuh/wazuh-qa/pull/2540/commits/1c627ab09b295462b74b4591310c2df8e20ee88f